index.js

```diff
@@ -386,13 +391,15 @@ window.mobileCheck = function() {
 
     function CreateTable(bCallAlarm = false /*Show notification, and sound alarm*/, bAutoScroll = true /*Auto scroll down to results table*/, bShowScrollNotif = true/*Show notification that you may have to scroll*/)
     {
-        let tableColumns = [{data: 'name', title: 'Centre name'}];
+        let tableColumns = [{data: 'name_district', title: 'Centre name'}];
```

Comment:
```suggestion
        let tableColumns = [{data: 'name', title: 'Centre name'}, render: function(data,type){return data.name + '(' +data.district + ')';}];
```
The data that you stored above - name and district will be available here in this functions data object. Whatever this render functions returns will be rendered in the corresponding cell of the table. This function can return string, html string, etc.

eg. if you want to make district in bold you can change this code further to:
```suggestion
        let tableColumns = [{data: 'name', title: 'Centre name'}, render: function(data,type){return data.name + '(<b>' +data.district + '</b>)';}];
```